File: CivitasR/lib/prueba.rb
#encoding:utf-8
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require_relative "civitas_Juego"
require_relative "vista_textual"
require_relative "controlador"


module Civitas
  class Prueba
    
    def main
      
      jugador1 = "David"
      jugador2 = "Ramón"

      jugadores = Array.new

      jugadores.push(jugador1)
      jugadores.push(jugador2)

      juego = Civitas_Juego.new(jugadores)
      vista = Vista_Textual.new()
      dado = Dado.instance()
      dado.set_debug(true)
      controlador = Controlador.new(juego, vista)
      controlador.juega()
      
    end
  end
  
  Test_P3 = Prueba.main
  
end

File: CivitasJ/src/civitas/Prueba.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

import java.util.ArrayList;
import juegoTexto.*;

/**
 *
 * @author ramonjtorres
 */
public class Prueba {
    
    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
    
        String jugador1 = "David";
        String jugador2 = "Ramón";

        ArrayList<String> jugadores = new ArrayList();

        jugadores.add(jugador1);
        jugadores.add(jugador2);

        CivitasJuego juego = new CivitasJuego(jugadores);
        VistaTextual vista = new VistaTextual();
        Dado dado = Dado.getInstance();
        dado.setDebug(true);
        Controlador controlador = new Controlador(juego, vista);
        controlador.juega();
    }
}
